README.md

```diff
@@ -1,6 +1,6 @@
 # 👨‍⚖️ jstz
 
-`jstz` (pronouced: "justice") is a JavaScript server runtime that powers Tezos 2.0 that uses [Boa](https://boajs.dev/) and is built in [Rust](https://www.rust-lang.org/).
+`jstz` (pronouced: "justice") is a JavaScript runtime powered by Tezos Smart Optimistic Rollups that uses [Boa](https://boajs.dev/) and is built in [Rust](https://www.rust-lang.org/).
```

Comment:
Are 'Smart Optimistic Rollups' a product with that capitalization, or should it be:
```suggestion
`jstz` (pronouced: "justice") is a JavaScript runtime powered by Tezos smart optimistic rollups that uses [Boa](https://boajs.dev/) and is built in [Rust](https://www.rust-lang.org/).
```